node-packages/commons/src/tasks.js

```diff
@@ -394,7 +408,100 @@ async function createRemoveTask(removeData: Object) {
 
   switch (project.activeSystemsRemove) {
     case 'lagoon_openshiftRemove':
-      return sendToLagoonTasks('remove-openshift', removeData);
+    case 'lagoon_openshiftBuildDeploy':
```

Comment:
I think this `case 'lagoon_openshiftBuildDeploy':` is too much here, it will never end up here, as `project.activeSystemsRemove` is never defined as this.